app/jinja_filters.py

```diff
@@ -647,22 +649,67 @@ def map_list_collector_config(
                 }
             )
 
-        rows.append(
-            {
-                "rowItems": [
-                    {
-                        "iconType": icon,
-                        "actions": actions,
-                        "rowTitle": item_name,
-                        "id": list_item.get("list_item_id"),
-                        "rowTitleAttributes": {
-                            "data-qa": f"list-item-{index}-label",
-                            "data-list-item-id": list_item.get("list_item_id"),
-                        },
-                    }
-                ]
-            }
-        )
+        if answer_title:
```

Comment:
There is some repetition/duplication in this method that could be refactored

Reply:
I removed repetition wherever we could spot it in this file now, thanks!